rune/libenclave/runelet.go

```diff
@@ -102,6 +103,14 @@ func StartInitialization() (exitCode int32, err error) {
 	initPipe.Close()
 	os.Unsetenv("_LIBENCLAVE_INITPIPE")
 
+	// If runelet run as detach mode, close logrus after initpipe closed.
+	envDetach := os.Getenv("_LIBENCLAVE_DETACH")
+	detach, err := strconv.Atoi(envDetach)
+	if detach != 0 {
+		logrus.SetOutput(ioutil.Discard)
+	}
+	os.Unsetenv("_LIBENCLAVE_DETACH")
+
```

Comment:
There is a small race window after initPipe.Close(). So move this chunk of code before initPipe.Close().